src/SignaturePad.Forms.WPF/SignaturePad.Forms.WPF.csproj

```diff
@@ -0,0 +1,46 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>net471</TargetFramework>
```

Comment:
Could we use the same version that forms targets? v4.6.1?

Reply:
you mean support both 471 and 461? Why not?

Reply:
maybe just v4.6.1 as there is no NEED for v4.7+. Also, with v4.7.1 we will have to install the targeting pack as this does not come by default - as our CI bot just failed

Reply:
changed to 461